backend/src/httpserver/httpserver/settings.py

```diff
@@ -0,0 +1,118 @@
+"""
+Django settings for httpserver project.
+
+Generated by 'django-admin startproject' using Django 4.1.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-+twskqgt5oq5#g7ar1w4jac7ajfmamsi^xx#7nl%svqadp)262"
```

Comment:
Should this be posted to the repository directly?

Reply:
Based on how I'm handling hosting this should not be a problem